src/evaluation/stats.jl

```diff
@@ -170,3 +172,70 @@ function aggregate_stats(df::DataFrame, criterion_col=:val_auc;
 	end
 	vcat(results...)
 end
+
+
+"""
+aggregate_stats_max_mean(df::DataFrame, criterion_col=:val_auc; undersample=Dict("ocsvm" => 100))
```

Comment:
Change the docstring to downsample (in the original aggregate stats its missing as well).